### ISSUE TITLE
There is an error with the GPG key and the Cuda home cannot be found

### ISSUE THREAD

**Issue by marziehphi** [georghess/voxel-mae#3]
A GPG key error occurred when I tried to install things using Dockerfile.

```bash
#0 3.621 W: GPG error: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A4B469963BF863CC
#0 3.621 E: The repository 'https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease' is not signed.
------
failed to solve: executor failed running [/bin/sh -c apt-get update && apt-get install -y ffmpeg libsm6 libxext6 git ninja-build libglib2.0-0 libsm6 libxrender-dev libxext6     && apt-get clean     && rm -rf /var/lib/apt/lists/*]: exit code: 100
```

**Comment by georghess** [comment]
Thanks for fixing this and opening a PR :)

**Comment by Varatharajan-Raja** [comment]
Exactly the same error was thrown again with the updated docker file. But I resolved it from [Sanjaas Comment](https://github.com/NVIDIA/nvidia-container-toolkit/issues/258)
```

RUN rm /etc/apt/sources.list.d/cuda.list
RUN rm /etc/apt/sources.list.d/nvidia-ml.list
RUN apt-key del 7fa2af80
RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub
RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/7fa2af80.pub
```

**Comment by georghess** [comment]
Sorry, I assumed NVIDIA/nvidia-docker#4 would have fixed this. Is that not the case?

**Comment by Varatharajan-Raja** [comment]
I'm not sure if this is user specific. But, Yes it did throw the same GPG Key error. So I had to remove the source files 1st and fetch the keys again. Maybe this helps someone. 
```

W: GPG error: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A4B469963BF863CC
E: The repository 'https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease' is not signed.
```

**Comment by georghess** [comment]
Okay, thanks for pointing this out. I've updated the Dockerfile in commit ef6040e.
